packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts

```diff
@@ -562,7 +605,75 @@ describe('TransferDomain', () => {
         }
       ])
       await expect(promise).rejects.toThrow(RpcApiError)
-      await expect(promise).rejects.toThrow('Non-DAT or LP tokens are not supported for transferdomain')
+      await expect(promise).rejects.toThrow(
+        'Non-DAT or LP tokens are not supported for transferdomain'
+      )
+    })
+
+    it('(evm -> dvm) should fail if LP token is transferred', async () => {
+      await createToken(container, 'DBTC')
+      const poolpairsBefore = await client.poolpair.listPoolPairs()
+      const poolpairsLengthBefore = Object.keys(poolpairsBefore).length
+      const address = await container.call('getnewaddress')
+
+      const metadata = {
+        tokenA: 'DFI',
+        tokenB: 'DBTC',
+        commission: 1,
+        status: true,
+        ownerAddress: address
+      }
+      const data = await client.poolpair.createPoolPair(metadata)
+      expect(typeof data).toStrictEqual('string')
+
+      await container.generate(1)
+      const poolpairsAfter = await client.poolpair.listPoolPairs()
+      expect(Object.keys(poolpairsAfter).length).toStrictEqual(
+        poolpairsLengthBefore + 1
+      )
```

Comment:
```suggestion
   
```
No need to assert tests in the creation of poolpair as it's already tested in other places

Reply:
removed

Reply:
removed